source/speech/__init__.py

```diff
@@ -1736,14 +1878,19 @@ def getFormatFieldSpeech(attrs,attrsCache=None,formatConfig=None,reason=None,uni
 	if attrsCache is not None:
 		attrsCache.clear()
 		attrsCache.update(attrs)
-	return separator.join(textList)
+	return textList
+
 
-def getTableInfoSpeech(tableInfo,oldTableInfo,extraDetail=False):
+def getTableInfoSpeech(
+		tableInfo, oldTableInfo, extraDetail=False

```

Comment:
Could you please type annotate these while at it? It's not directly clear to me what type is expected here.